asapdiscovery-data/asapdiscovery/data/tests/aws/test_aws_cli.py

```diff
@@ -0,0 +1,64 @@
+import contextlib
+import os
+import traceback
+import uuid
+
+from asapdiscovery.data.cli import cli
+from click.testing import CliRunner
+
+
+def click_success(result):
+    if result.exit_code != 0:  # -no-cov-  (only occurs on test error)
+        print(result.output)
+        traceback.print_tb(result.exc_info[2])
+        print(result.exc_info[0], result.exc_info[1])
+    return result.exit_code == 0
+
+
+# based on https://stackoverflow.com/a/34333710
+@contextlib.contextmanager
+def set_env_vars(env):
+    old_env = dict(os.environ)
+    try:
+        os.environ.update(env)
+        yield
+    finally:
+        os.environ.clear()
+        os.environ.update(old_env)
+
+
+class TestS3:
```

Comment:
```suggestion
class TestS3CLI:
```
to avoid collision with the `TestS3` class in `test_aws_s3`